src/pages/installation.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: ~/layouts/Main.astro
+---
+
+# Installing Astro
+
+Welcome to Astro, we are really excited that you are here and would like to assist you in getting up and started with Astro in no time.
+
+<!--
+    Quick Install
+    Manual Install
+    Troubleshooting
+ -->
+
+## Preflight
+
+Before we begin, let us check to make sure that everything is in order.
+
+To use Astro you need to have Node.js already installed and working on your machine. If you haven't got Node.js installed, you can download it from their site.
+
+⚠️ **Important**: Astro is built and uses ESM modules as default. This is only supported from Node versions 12 and above,. However, to use Astro you would need to have at least version `14.16.1` installed on your machine.
+
+You would also need a package manager to install and use Astro.
+
+Node.js ships their own package manager during installation, so you would have **npm** on your system. You are not restricted to using only npm, you can use your preferred package manager if you so wish:
+    - [npm](https://docs.npmjs.com/about-npm)
+    - [Yarn](https://yarnpkg.com/getting-started)
+    - [pnpm](https://pnpm.io/installation)
+
+We ask that your table and chairs and in upright postilions, and you remain seated until after we take-off.
+
+Lastly remember, "Don't Panic, and carry a towel."
+
+That is it for the preflight checks, now we can get you started.
+
+## Quickstart
+
+To quickly get up and running with Astro, simply enter the following commands into your terminal
+
+```bash
+# create your project
+mkdir new-project-directory
+cd new-project-directory
+npm init astro || yarn add astro
+
+# install your dependencies
+npm install || yarn
+
+# start the dev server and open your browser
+npm run start || yarn start
+```
+
+👍 You are now good to go with Astro...
+
+##   Astro Setup
+
+You can install Astro via the **CLI**. This way you can kick off with some starter projects or view some demo projects to get better acquainted with Astro.
+
+First create your directory that we would be deploying Astro in.
+
+```bash
+mkdir my-astro-project
+```
+
+Inside your directory, we would first install Astro using your preferred package manager:
+
+```bash
+npm init astro || yarn add astro
+```
+
+You would be asked if you wish to download the Astro packages to the System, enter `Y` to accept.
+
+Next our **CLI tool** would start up to walk you through the rest of installation.
+
+It would ask you which *template* you wish to use, here we have a few options for you to choose from to get started.
+
+- **Starter Kit** - This is our base project that allows you to take Astro and start building anything you wish
```

Comment:
@FredKSchott maybe we could have the base templates deployed to add a link here as a reference?

Reply:
Oh I like that. That's out of scope for now (we'd need to redeploy ever template after every commit to keep them up to date) but if you want to spin up an issue that could be great.

OR, just add a CodeSandbox/StackBlitz link to every project README and then add that here as well.

Reply:
No problems, I'll work on making up the sandboxed for each project over the weekend, and have them linked into the page, quick question, is there an astro user account that I for anyone of those codeboxes that I could prehaps use to set these up, It would make sense to keep all branding and user pointing reference material to be on point and consistent when off page,

Reply:
I believe there is no astro account, and being sincere I don't see the need. You can just add Astro as a tag to the template.

There are already some available. Maybe we could just decide on a prefix. like `@astro/` not sure if `@astrojs/` makes more sense.
![image](https://user-images.githubusercontent.com/11237366/123334958-19aa0f80-d544-11eb-87ee-fe0f6d30bc01.png)